### PR TITLE
[BEAM-2284] Completely removes OldDoFn from Beam.

### PR DESCRIPTION
**Pull request by jkff**
While doing this, noticed that WindowingInternals can also be moved into Dataflow worker and deleted from Beam. Will do that separately.

R: @tgroh

### CODE REVIEW

**Comment by jkff** [comment]
Run Dataflow ValidatesRunner

**Comment by jkff** [comment]
Run Dataflow ValidatesRunner

**Comment by jkff** [comment]
retest this please

**Comment by jkff** [comment]
Ugh flaky tests.

**Comment by jkff** [comment]
retest this please

**Comment by jkff** [comment]
run Dataflow ValidatesRunner

**Comment by jkff** [comment]
CC: @kennknowles

File: runners/core-java/src/main/java/org/apache/beam/runners/core/GroupAlsoByWindowsAggregators.java [hunk 1]
```diff
@@ -17,23 +17,10 @@
...
- * @param <K> key type
- * @param <InputT> input value element type
- * @param <OutputT> output value element type
- * @param <W> window type
+ * Common aggregator names for {@link GroupByKeyViaGroupByKeyOnly.GroupAlsoByWindow}.
  */
-@SystemDoFnInternal
-public abstract class GroupAlsoByWindowsDoFn<K, InputT, OutputT, W extends BoundedWindow>
-    extends OldDoFn<KV<K, Iterable<WindowedValue<InputT>>>, KV<K, OutputT>> {
+public abstract class GroupAlsoByWindowsAggregators {
```
Comment (kennknowles) [inline runners/core-java/src/main/java/org/apache/beam/runners/core/GroupAlsoByWindowsAggregators.java]: These are metrics now. I wouldn't block this PR on the rename, but feel free to do the rename and still merge.

**Pull request closed without merging**